### PR TITLE
[INLONG-409][Sort] Add guide doc for the mongodb-cdc connector

### LINKED ISSUE #409: [Sort] Add document for mongodb-cdc extract node
Add document for mongodb-cdc extract node

### PR DESCRIPTION
**Pull request by Oneal65**
Fixes #409 

### Motivation

add md for mongodb-cdc

### Modifications
add documents about mongodb extract node
### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/data_node/extract_node/mongodb-cdc-en.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+| database_name   | STRING NOT NULL           | Name of the database that contain the row.                   |
+| collection_name | STRING NOT NULL           | Name of the collection that contain the row.                 |
+| op_ts           | TIMESTAMP_LTZ(3) NOT NULL | It indicates the time that the change was made in the database. If the record is read from snapshot of the table instead of the change stream, the value is always 0. |
+
+
+The extended CREATE TABLE example demonstrates the syntax for exposing these metadata fields:
+```sql
+CREATE TABLE `mysql_extract_node` (
+    db_name STRING METADATA FROM 'database_name' VIRTUAL,
+    table_name STRING METADATA  FROM 'table_name' VIRTUAL,
```
Comment (gong) [inline docs/data_node/extract_node/mongodb-cdc-en.md]: `table_name` should be `collection_name`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc-ch.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+| database_name   | STRING NOT NULL           | 包含该行的数据库的名称。                                     |
+| collection_name | STRING NOT NULL           | 包含该行的集合的名称。                                       |
+| op_ts           | TIMESTAMP_LTZ(3) NOT NULL | 它指示在数据库中进行更改的时间。<br/>如果记录是从表的快照而不是更改流中读取的，则该值始终为 0。 |
+
+
+扩展的 CREATE TABLE 示例演示了公开这些元数据字段的语法：
+```sql
+CREATE TABLE `mysql_extract_node` (
+    db_name STRING METADATA FROM 'database_name' VIRTUAL,
+    table_name STRING METADATA  FROM 'table_name' VIRTUAL,
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc-ch.md]: `tabe_name` should be `collection_name`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc-ch.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    price ROW<amount DECIMAL(10,2), currency STRING>, -- embedded document
+    suppliers ARRAY<ROW<name STRING, address STRING>>, -- embedded documents
+    PRIMARY KEY(_id) NOT ENFORCED
+) WITH (
+      'connector' = 'mongodb-cdc', 
+      'hostname' = 'YourHostname',
+      'username' = 'YourUsername',
+      'password' = 'YourPassword',
+      'database-name' = 'YourDatabase',
+      'table-name' = 'YourTable'
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc-ch.md]: `database-name` should be `database`.
`table-name` should be `collection`

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md [hunk 4]
```diff
@@ -20,10 +20,10 @@ I.为了设置 MongoDB CDC 连接器，下表提供了使用构建自动化工
 
 ```xml
 <dependency>
-  <groupId>com.ververica</groupId>
-  <artifactId>flink-connector-mongodb-cdc</artifactId>
-  <!-- the dependency is available only for stable releases. -->
-  <version>2.1.1</version>
+    <groupId>org.apache.inlong</groupId>
+    <artifactId>sort-connector-mongodb-cdc</artifactId>
+    <!-- Choose the version that suits your application -->
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md:25]: translate it

**Review by healchow — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md [hunk 5]
```diff
@@ -1,4 +1,192 @@
...
+---
+
+## MongoDB-CDC Extract 节点
+
+MongoDB CDC 连接器允许从 MongoDB 读取快照数据和增量数据。本文档介绍如何设置 MongoDB CDC 连接器以对 MongoDB 运行 SQL 查询。
+
+## 支持的版本
+| Extract 节点                     | 版本                                          |
+| ------------------------------- | --------------------------------------------- |
+| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): `>=` 3.6 |
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md:13]: [mongodb-cdc] -> [MongoDB-CDC]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md [hunk 6]
```diff
@@ -1,4 +1,192 @@
...
+
+### InLong Dashboard 用法
+
+TODO: 未来会支持
+
+### InLong Manager 用法
+
+TODO: 未来会支持
+
+## MongoDB Extract 节点选项
```
Comment (yunqingmoswu) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md:115]: 选项 -> 参数

File: docs/data_node/extract_node/mongodb-cdc.md [hunk 7]
```diff
@@ -1,4 +1,197 @@
...
+---
+
+## MongoDB-CDC Extract Node
+
+The MongoDB CDC connector allows for reading snapshot data and incremental data from MongoDB. This document describes how to setup the MongoDB CDC connector to run SQL queries against MongoDB.
+
+## Supported Version
+| Extract Node                    | Version                                      |
+| ------------------------------- | -------------------------------------------- |
+| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): `>=` 3.6 |
```
Comment (yunqingmoswu) [inline docs/data_node/extract_node/mongodb-cdc.md:13]: `>=` -> `&gt;=`

File: docs/data_node/extract_node/mongodb-cdc.md [hunk 8]
```diff
@@ -10,20 +10,20 @@ The MongoDB CDC connector allows for reading snapshot data and incremental data
 ## Supported Version
 | Extract Node                    | Version                                      |
 | ------------------------------- | -------------------------------------------- |
-| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): \>= 3.6 |
+| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): `>=` 3.6 |
```
Comment (gong) [inline docs/data_node/extract_node/mongodb-cdc.md:13]: `mongodb-cdc` change to `MongoDB-CDC`. keep consistent with title

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md [hunk 9]
```diff
@@ -3,14 +3,14 @@ title: MongoDB-CDC
...
 
 MongoDB CDC 连接器允许从 MongoDB 读取快照数据和增量数据。本文档介绍如何设置 MongoDB CDC 连接器以对 MongoDB 运行 SQL 查询。
 
 ## 支持的版本
-| Extract节点                     | 版本                                         |
-| ------------------------------- | -------------------------------------------- |
-| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): \>= 3.6 |
+| Extract节点                     | 版本                                          |
+| ------------------------------- | --------------------------------------------- |
+| [mongodb-cdc](./mongodb-cdc.md) | [MongoDB](https://www.mongodb.com/): `>=` 3.6 |
```
Comment (gong) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/extract_node/mongodb-cdc.md:13]: `mongodb-cdc` change to `MongoDB-CDC`

**Review by yunqingmoswu — APPROVED** [review]

**Review by gong — APPROVED** [review]

**Pull request merged**